group_vars/model_ubnt_usw_flex.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+# factory flash instructions
+# https://github.com/openwrt/openwrt/commit/a9839697896c4fdf8c44a06bbce466ce52493069
+
+openwrt_version: snapshot
+target: ramips/mt7621
+
+model__packages__to_merge:
+  - "-tcpdump"       # save space, our partition is only 7360k, not 8m
```

Comment:
I would rather prefer to set the low_flash flag, instead of manually dropping packages.

See
https://github.com/freifunk-berlin/bbb-configs/blob/d7c9690fd998df83039abc753502115cade76757/roles/cfg_openwrt/tasks/conditional_packages.yml#L29